app/docs/0.13.x/clustering.md

```diff
@@ -149,9 +149,8 @@ Now, say we add a plugin to this Service via node `A`'s Admin API:
 
 ```bash
 # node A
-$ curl -X POST http://127.0.0.1:8001/plugins \
+$ curl -X POST http://127.0.0.1:8001/services/example/plugins \
```

Comment:
I am only noticing now that is example is skewed.

1. The above Route creation warrants a comment saying something like "we could use /routes but /services/:service/routes is a shortcut
2. It creates `paths[]=example.com` which is incorrect
3. "example" as a name is confusing, because it conflicts in this endpoint with the previous `:8000/example` requests. I like "example-service" better as used in other occurrences in this patch

Reply:
I used `example` because that was the name of the Service being created above. Can change both.